package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "on-chain-identity-gateway",
+  "version": "1.0.0",
+  "description": "Welcome to the Identity.com On-chain Identity Gateway monorepo.",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "dependencies": {
+    "husky": "^8.0.1"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "prepare": "husky install",
+    "program_v2_lint": "cd solana/program_v2 && yarn lint"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": ""
+    }
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/identity-com/on-chain-identity-gateway.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
Can you double check all licenses in the Repo? I think they are all MIT, therefore it should be the same here.

Reply:
fixed